pco/src/chunk_config.rs

```diff
@@ -46,6 +46,20 @@ pub enum FloatMultSpec {
   // TODO support a LossyEnabled mode that always drops the ULPs latent var
 }
 
+/// Configures whether float-decomposition detection is enabled.
+///
+/// Examples where this helps:
+/// * float-valued data stored in a type that is unnecessarily wide (e.g. stored
+///   as `f64`s where only the first 32 bits are used)
+pub enum FloatDecompSpec {
```

Comment:
Let's call this `QuantizeSpec` or `FloatQuantizeSpec`?

Reply:
Will do.  Saving the mass-rename until the other stuff is resolved because mass-renames in git are a pain.

Reply:
Done.